nexus/types/src/external_api/params.rs

```diff
@@ -1034,6 +1038,12 @@ pub struct InstanceCreate {
     pub start: bool,
 }
 
+/// Parameters of an `Instance` that can be reconfigured after creation.
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+pub struct InstanceUpdate {
```

Comment:
so, my thinking is that for names like this, they're part of the public API (in `openapi/nexus.json` etc), so it's much more valuable to be consistent - this is more in line with items like `InstanceNetworkInterfaceUpdate`, `ProjectUpdate`, `SiloQuotasUpdate`, or `VpcFirewallRuleUpdate`. going with `InstanceReconfigure` would make this one item different because we happen to have a different name for an internal operation and that seems.. not great especially when we're thinking about changing it already?

(this is also why it's `update` in `external-api/src/lib.rs`: at that point i expect readers to be approaching with the external name in mind, so that's what you'd be searching for)

Reply:
Agree. "Update" for consistency.

Reply:
+1 on `Update` for consistency